docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -238,6 +238,8 @@ Before you can run the application locally, you will need to start the MongoDB s
 > Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
 >
 > If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.
+>
+>If you are using gitpod you may skip starting mongo as it should already be running
```

Comment:
I suggest we remove this in favour of adjusting this line in the `use-a-local-machine-or-gitpod` section:

> That's it, you can now skip to the 'syncing up from parent' section after you have launched a GitPod workspace.

To

> That's it, you can now skip to the [follow best-practices section](/codebase-best-practices?id=codebase-best-practices) after you have launched a Gitpod workspace.

```suggestion
```

Reply:
Sweet. That works cause then they would know what they can skip